CodeFirst/Assement.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TechnicalAssesment
+{
+    public enum GameResult
+    {
+        GameNotStarted,
+        DealerWon,
+        PlayerWon,
+        Tie,
+    }
+
+    public class ScoreBoard
+    {
+        public byte playerWins;
+        public int DealerWins;
+        public byte Ties { get; set; }
+    }
+
+    public class Deck
+    {
+        public List<int> cards;
+        public int MaxCards = 52;
+        public Deck()
+        {
+            cards = new List<int>();
+        }
+
+        public void Shuffle()
+        {
+            for (int i = 0; i < MaxCards; i++)
+            {
+                cards.Add(i);
+            }
+        }
+
+        public int DrawCard()
+        {
+            Random r = new Random();
+            int index = r.Next(MaxCards);
+            int selectedCard = cards[index];
+            cards.RemoveAt(index);
+            return selectedCard;
+        }
+
+        public GameResult determineWinResult(int PlayerCard, int DealerCard)
+        {
+            GameResult result = new GameResult();
+            if (PlayerCard % 13 > DealerCard % 13)
+            {
+                result = GameResult.PlayerWon;
+            }
+            else if (PlayerCard % 12 < DealerCard % 13)
+            {
+                result = GameResult.DealerWon;
+            }
+            else
+                result = GameResult.Tie;
+            return result;
+        }
+    }
+
+    public class Game
+    {
+        public ScoreBoard ScoreBoard;
+        public Deck deck;
+
+        private byte DealerCard;
+        private byte PlayerCard;
```

Comment:
No need to for class declaration as the values are changed each draw and can be I main game logic method